tests/schema/tests.py

```diff
@@ -517,6 +517,15 @@ def test_add_field(self):
             'column': editor.quote_name(new_field.name),
         }
         self.assertFalse(any(drop_default_sql in query['sql'] for query in ctx.captured_queries))
+        # Table is not rebuilt.
+        self.assertIs(any(
+            'CREATE TABLE' in query['sql']
+            for query in ctx.captured_queries
+        ), False)
+        self.assertIs(any(
+            'DROP TABLE' in query['sql']
+            for query in ctx.captured_queries
+        ), False)
```

Comment:
why not `assertFalse` ?

```suggestion
        self.assertFalse(any(
            'CREATE TABLE' in query['sql']
            for query in ctx.captured_queries
        )
        self.assertFalse(any(
            'DROP TABLE' in query['sql']
            for query in ctx.captured_queries
        ))
```

Reply:
`assertFalse()` matches anything falsy. If you want to check for `False` only you need to use `assertIs(…, False)`.

Reply:
But `any()` always returns a bool? https://docs.python.org/3/library/functions.html#any